fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerSimpleTest.java

```diff
@@ -901,6 +901,26 @@ public void testConfigLoader() throws Exception {
         }
     }
 
+    @Test
+    void testConfigLoaderInitialLimit() throws Exception {
+        final Index index = new Index("newIndex", field("num_value_unique"));
+        populateData(40);
+
+        final FDBRecordStoreTestBase.RecordMetaDataHook hook = allIndexesHook(List.of(index));
+
+        openSimpleMetaData(hook);
+
+        final FDBStoreTimer timer = new FDBStoreTimer();
+        try (OnlineIndexer indexBuilder = newIndexerBuilder(index, timer)
+                .setInitialLimit(4)
+                .setLimit(10000)
+                .setConfigLoader(old -> old)
+                .build()) {
+            indexBuilder.buildIndex();
```

Comment:
What is this testing?

Reply:
It was testing the enforcement of initial limit during every config load.

Reply:
Added comment..